### DISCUSSION TITLE
Half screen black

### DISCUSSION THREAD

**Discussion by gilforge** [mulaRahul/keyviz#1] [General]
Hello. It's an excellent software but for the moment, a huge black rectangle covers the bottom half of my screen, making any demonstration impossible.
windows 10 Family 20H2, non activated (at this time).

Best regards :)

**Comment by mulaRahul** [comment]
The black rectangle is probably the application window. It should be
transparent but when windows can't render it transparent, it becomes black.
Please open an issue on the repository with relevant information and
images, if you can.
